website/content/en/preview/getting-started/scripts/add-provisioner.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I'm not sure it's necessary to break these into scripts to add a provisioner or install grafana. What's the use-case here?   The main reason for this is just to make sure the getting started guide works and could guide you through a one-click setup. I would envision that one click setup to include grafana. Any additional provisioners would be manual by the user. I don't think we'd want to help through these scripts after we get the initial karpenter cluster up and running.